packages/block-library/src/navigation/edit/use-convert-classic-menu-to-block-menu.js

```diff
@@ -21,15 +20,7 @@ export const CLASSIC_MENU_CONVERSION_IDLE = 'idle';
 // do not import the same classic menu twice.
 let classicMenuBeingConvertedId = null;
 
-function useConvertClassicToBlockMenu( clientId ) {
-	/*
-	 * The wp_navigation post is created as a draft so the changes on the frontend and
-	 * the site editor are not permanent without a save interaction done by the user.
-	 */
-	const { create: createNavigationMenu } = useCreateNavigationMenu(
-		clientId,
-		'draft'
-	);
+function useConvertClassicToBlockMenu( clientId, createNavigationMenu ) {
```

Comment:
`clientId` is now unused?
```suggestion
function useConvertClassicToBlockMenu( createNavigationMenu ) {
```

You'd need to amend all calls to the hook.

Reply:
I've gone ahead and actioned this for you.